install/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Invalid number of parameters"
+		exit 1
+fi
+
+JSONPATH1="{.spec.replicas}"
+JSONPATH2="{.status.readyReplicas}"
+ZkOprName=zoo-op
```

Comment:
Bash scripting naming conventions suggest variable names should be all in lower case, unless they are environment variables that need to be exported. Those should be all in upper case.
Please change variable names accordingly.